docs/howto/handle_pagination.md

```diff
@@ -0,0 +1,75 @@
+[Back to Guides](../README.md)
+
+# How to handle pagination
+
+## JSON API adapter
+
+If you want to paginate a collection with the `json_api` adapter, you can either use the top-level `links` member to provide pagination links or the top-level `meta` member to provide pagination information:
```

Comment:
I already reference the link documentation below. I don't see why we should put `meta` in the `links` documentation though. That's two different things in my opinion.